pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "prometheus-eaton-ups-exporter"
+authors = [
+    {name = "Mathis Lövenich"},
+]
+description = "A Prometheus exporter for Eaton UPSs."
+version = "1.2.0"
+readme = "README.md"
+requires-python = ">=3.11"
+keywords = ["prometheus", "exporter", "eaton"]
+license = {text = "ISC"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+]
+dependencies = [
+    "prometheus_client",
+    "requests",
+    "urllib3",
```

Comment:
Out of scope, but this dependency is used in one place, and simply to disable SSL verification.

`requests` supports this feature directly. https://requests.readthedocs.io/en/latest/user/advanced/#ssl-cert-verification

Reply:
I created an issue based on this, and will look into that in a future PR.